manifests/d/DuckStudio/GitHubLabelsManager/1.1/DuckStudio.GitHubLabelsManager.installer.yaml

```
@@ -0,0 +1,43 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubLabelsManager
+PackageVersion: "1.1"
```

```suggestion
PackageVersion: "v1.1"
```

This should match control panel exactly, as version prefixes are not trimmed in the 1.8 client. After making this change, the version folder will need to be renamed